src/automerge.py

```diff
@@ -90,21 +90,13 @@ def run_git_command(command_args: str) -> subprocess.CompletedProcess:
 for pr in automerge_prs:
     base_branch = repo.get_branch(pr.base.ref)
     if base_branch.protected:
-        required_status_checks = base_branch.get_required_status_checks()
-        latest_commit = pr.get_commits().reversed[0]
-        latest_commit_checks = {check_run.name: check_run for check_run in latest_commit.get_check_runs()}
-        all_checks_passed = True
-        for required_check in required_status_checks.contexts:
-            if required_check not in latest_commit_checks:
-                print(f"Required check {required_check} is missing in the latest commit.")
-                all_checks_passed = False
-            else:
-                check_run = latest_commit_checks[required_check]
-                if check_run.conclusion == 'success':
-                    print(f"Required check {required_check} passed on PR#{pr.number}")
-                else:
-                    print(f"Required check {required_check} failed or is pending on PR#{pr.number}")
-                    all_checks_passed = False
+        commit = pr.get_commits().reversed[0]
+        statuses = commit.get_statuses()
+        all_checks_passed = all(print(status) for status in statuses)
```

Comment:
This evaluates to `False` regardless of `statuses`. (And as a side effect, will print the first `status`.)

Reply:
The tests came back as expected. 
https://github.com/runtimeverification/automerge/actions/runs/12061856900/job/33634749102#step:4:1

I'll debug this now. Must be getting a false positive

Reply:
Addressed this issue. This required reworking permissions to introduce the use for a Github App. 
Remaining is to update the README and docs.